__tests__/methodDoc-test.ts

```diff
@@ -0,0 +1,63 @@
+import * as api from '../src/pslLint/api';
+import { MethodDocumentation, MethodSeparator, TwoEmptyLines } from '../src/pslLint/methodDoc';
+import * as utils from './ruleUtils';
+
+function messageOnLine(lineNumber: number, allDiagnostics: api.Diagnostic[]): string {
+	const diagnosticsOnLine = utils.diagnosticsOnLine(lineNumber, allDiagnostics);
+	if (!diagnosticsOnLine.length) return '';
+	return diagnosticsOnLine[0].message;
+}
+
+describe('Parameter tests', () => {
+	const procName = 'ZMethodDoc.PROC';
+	let docDiagnostics: api.Diagnostic[] = [];
+	let emptyLineDiagnostics: api.Diagnostic[] = [];
+	let separatorDiagnostics: api.Diagnostic[] = [];
+
+	beforeAll(async () => {
+		docDiagnostics = await utils.getDiagnostics(procName, MethodDocumentation.name);
+		emptyLineDiagnostics = await utils.getDiagnostics(procName, TwoEmptyLines.name);
+		separatorDiagnostics = await utils.getDiagnostics(procName, MethodSeparator.name);
+	});
+
+	test('allProblemsNoLineAbove', () => {
+		// expect(messageOnLine(0, docDiagnostics)).toBe('TODO');
```

Comment:
Do you still need these comments?

Reply:
They're placeholders to show how to write the tests. They will be uncommented and updated soon.